### ISSUE TITLE
[Bug]: Custom sound on Android plays only after first reboot

### ISSUE THREAD

**Issue by websystems-dev** [OneSignal/OneSignal-Cordova-SDK#841]
### What happened?

After first installation, notifications does not play the custom sound and is silent.
After a reboot, notification sound is ok.
I'm using Ionic Framework with Capacitor.
- onesignal-cordova-plugin@3.3.0
- @awesome-cordova-plugins/onesignal@6.2.0

### Steps to reproduce?

```Markdown
- I tried multiple combination and the problem persists:
-- app built in debug mode and in release mode
-- app installed directy with Android Studio and app downloaded from store (internal test channel)
-- Android with API 31/32/33
-- Device Pixel 4 and Nexus 5X

- I installed the app and send a test notification from Onesignal Dashboard (the notification contains info about channel and custom sound)
- The device receives the notifications but plays no sound
- I reboot the device from the OS
- After the reboot, the same notification plays normally with my custom sound.
- At this point, if I uninstall and reinstall the app, custom sound plays normally right after installation (no further reboot is required).
```

### What did you expect to happen?

Custom sound shoud play normaly after first installation

### OneSignal Cordova SDK version

 onesignal-cordova-plugin 3.3.0

### Which platform(s) are affected?

- [ ] iOS
- [X] Android

### Relevant log output

```Shell
I'm unable to spot something interesting in LogCat.
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jennantilla** [comment]
Hello @websystems-dev apologies that this issue was missed.

New sounds can take a while to propagate. Is it possible that the app was not updated to the latest version which contained your new sound resource? If you have an older version of your app without the sound resource and receive a notification that references it, you will hear only the default system notification sound.

Since this issue is quite old, I'm going to close it out. But if you are still having this problem, feel free to create a new issue and we can look into it more!

We also have a new major release with many new improvements and enhancements. Please see our [migration guide](https://github.com/OneSignal/OneSignal-Cordova-SDK/blob/main/MIGRATION_GUIDE.md) for full details on upgrading.

Thanks!
